lisp/forge-commands.el

```diff
@@ -893,6 +895,13 @@ is added anyway.  Currently this only supports Github and Gitlab."
 
 ;;; Misc
 
+(transient-define-infix forge-forge.max-topics ()
+  "Change the maximum number of topics to pull at once."
+  :class 'magit--git-variable
+  :variable "forge.max-topics"
+  :reader (lambda (&rest args)
+            (apply #'read-string args)))
```

Comment:
Is this really necessary? I think this should fall through to using `read-string` if no reader is given explicitly.

Reply:
You're spot on there :sweat_smile: Thanks